react-version/src/style/Game.scss

```diff
@@ -0,0 +1,260 @@
+.game {
+	display: flex;
+	justify-content: center;
+	padding: 20px 0 0 0;
+	transition: 1s;
+
+	.click-start-hex {
+		position: absolute;
+		display: flex;
+		justify-content: center;
+		align-items: center;
+		top: 600px;
```

Comment:
In general try to avoid such huge position/margin/padding values - instead use some relative units like % or something else. What if someone opens the game on smaller screen? Board won't be centered, it might be even invisible.

Using pixel and similar units makes sense when values are not higher than, lets assume < 100px (but thats not the rule!)